src/account/AccountCreateTransaction.ts

```diff
@@ -39,19 +40,19 @@ export class AccountCreateTransaction extends TransactionBuilder {
         return this;
     }
 
-    setInitialBalance(balance: number | BigNumber | Hbar): this {
-        this.body.setInitialbalance(toPositiveTinybarString(balance));
+    setInitialBalance(balance: Tinybar | Hbar): this {
+        this.body.setInitialbalance(tinybarToString(balance));
         return this;
     }
 
     setReceiveRecordThreshold(threshold: number): this {
-        checkNumber(threshold);
+        tinybarRangeCheck(threshold);
         this.body.setReceiverecordthreshold(threshold);
         return this;
     }
 
     setSendRecordThreshold(threshold: number): this {
```

Comment:
These two methods will have their signatures changed in my `AccountInfoQuery` PR.